### PR TITLE
 feat: input filename as argument to assertVersionBump

### PR DESCRIPTION
**Pull request by rodrigo4244**

### CODE REVIEW

**Comment by rodrigo4244** [comment]
To test it you can:
1. Run `node src/assertVersionBump.js manifest.json`
* See that it is going to raise an error. 
Then:
1. Add a new file (e.g `manifest.json`)
2. Add `{ "version": "1.0.0" }
3. Commit
4. Run `node src/assertVersionBump.js manifest.json`
* No error raised.

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "fix": "eslint --cache --fix scripts src test",
     "lint": "eslint --cache -f node_modules/eslint-formatter-pretty scripts src test",
     "pretest": "yarn run lint",
-    "test": "yarn run lint",
+    "test": "exit 0",
```
[the remark this answers is not in the document]
Comment (rodrigo4244) [reply package.json]: Because we don't need to lint two times.

File: src/assertVersionBump.js [hunk 2]
```diff
@@ -20,6 +20,8 @@ const {
   trimCharsStart,
 } = require('lodash/fp')
 
+const [ , , fileToAssertVersionBump = 'package.json' ] = process.argv
```
Comment (gunar) [inline src/assertVersionBump.js]: Just `fileName` would suffice, as context of this variable is a file called `assertVersionBump`.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (gunar) [reply package.json]: Agreed, but it should've been a separate commit (and PR maybe).

**Comment by gunar** [comment]
Am accepting this as is and you may decide whether to rename that variable or not.

**Pull request merged**